server/models/Profile.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require("mongoose");
+const { weeklyAvailability } = require("./availability");
+const { paymentProfile } = require("./paymentProfile");
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true
+  },
+  lastName: {
+    type: String,
+    required: true
+  },
+  gender: {
+    type: String,
+    enum: ["MALE", "FEMALE", "OTHER"],
+    required: true
```

Comment:
```suggestion
    required: true,
    uppercase: true,
```
You can add the uppercase validator here too :)

Reply:
Thank you. Done